### ISSUE TITLE
Issue about joint loss training

### ISSUE THREAD

**Issue by seekFire** [JimmySuen/integral-human-pose#43]
I noticed some words in your paper:
_**For integral regression methods (I1, I2, I3, and their multi-stage versions),
the network is pre-trained only using heat map loss (thus their H versions) and
then, only integral loss is used. We found this training strategy working slightly
better than training from scratch using both losses.**_
First, Do you mean training with heatmap loss at first and then using integral loss for subsequent training is slightly better than using both kinds of loss for training at the same time?
Second, I noticed the keyword **end_epoch** in the .yaml file of configuration in your repo. The values of them are almost hundreds. So whether it means training the pretrained model using integral loss with hundreds of epochs? Or just training from scratch?
